packages/template-retail-react-app/app/components/header/index.jsx

```diff
@@ -153,6 +162,7 @@ const Header = ({
                             id: 'header.button.assistive_msg.menu',
                             defaultMessage: 'Menu'
                         })}
+                        title="Opens a dialog"
```

Comment:
Is there any translation that we can reuse here so we don't have to reach out to the translation team? I'm assuming that the screen ready will read this text aloud?